atom/renderer/api/atom_api_spell_check_client.cc

```diff
@@ -37,6 +38,27 @@ bool HasWordCharacters(const base::string16& text, int index) {
 
 }  // namespace
 
+class SpellCheckClient::SpellcheckRequest {
```

Comment:
If this isn't being used just yet, do we want to leave a little documentation hint here?

Reply:
it is being used: check https://github.com/electron/electron/pull/12112/files#diff-bfd35a1e98d8d7cb08cc06502b38b7d8R110 and its description at https://github.com/electron/electron/pull/12112/files#diff-19a6eac471361938d1d72714b1774c09R97.